### PR TITLE
WIP: added logging

### PR DESCRIPTION
**Pull request by harryhmc**

### CODE REVIEW

File: coordinator/src/routes/prescription/post-convert-message.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default [
       (requestPayload: Bundle, request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
         const isSmokeTest = request.headers["x-smoke-test"]
         const contentType = isSmokeTest ? CONTENT_TYPE_PLAIN_TEXT : CONTENT_TYPE_XML
+        request.log("info", "Building HL7V3 message")
```
Comment (dan-jones-iw) [inline coordinator/src/routes/prescription/post-convert-message.ts:20]: got `request.logger.info()` above and `request.log("info", )` here, maybe best to choose one? think i saw the `.logger.info` in the other pr

File: coordinator/src/routes/health/get-status.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default [
       let validator = false
 
       try {
+        request.logger.info('Checking validator status')
```
Comment (dan-jones-iw) [inline coordinator/src/routes/health/get-status.ts:13]: not sure how best to make it look but could we get timestamps in these logs?

File: coordinator/src/routes/util.ts [hunk 3]
```diff
@@ -105,7 +105,9 @@ export function validatingHandler(handler: Handler<fhir.Bundle>) {
     if (request.headers["x-skip-validation"]) {
       request.log("info", "Skipping call to FHIR validator")
     } else {
+      request.log("info", "Making call to FHIR validator")
```
Comment (dan-jones-iw) [inline coordinator/src/routes/util.ts:108]: just a few left here :sweat_smile:

Comment (harryhmc) [reply coordinator/src/routes/util.ts:108]: oops ty for catching

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**